js/views/transactionModalVw.js

```diff
@@ -43,6 +43,9 @@ module.exports = baseVw.extend({
     'click .js-transactionShowContract': 'showContract',
     'click .js-transactionHideContract': 'hideContract',
     'click .js-acceptResolution': 'acceptResolution',
+    //'click .js-refundTransaction': 'showRefundOrder',
```

Comment:
Yes. This is for the code that will show a form to add a reason for the refund. The API doesn't currently support that, but it may later, so I want to leave it in.